Code/C-101/ReadInput/Challenges/circle.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+
+int main()
+{
+    int const pi = 3.14;
+    int radius, diameter, area, circumference;
+    printf("Welcome to Circle Calculator!");
+    printf("");
+    printf("%s", "What do you want your circle's radius to be?");
+    scanf("%d", &radius);
+
+    diameter = radius * 2;
+    area = pi * radius * radius;
+    circumference = 2 * pi * radius;
```

Comment:
Circumference can be pi * diameter here because, you are already calculating diameter.